server/src/main/java/io/deephaven/server/plugin/python/ObjectTypeAdapter.java

```diff
@@ -88,12 +105,21 @@ private PythonServerMessageStream(PyObject instance) {
         public void onData(ByteBuffer payload, Object[] references) {
             byte[] bytes = new byte[payload.limit()];
             payload.get(bytes);
+
+            for (int i = 0; i < references.length; i++) {
+                Object reference = references[i];
+                if (reference instanceof LivePyObjectWrapper) {
+                    reference = ((LivePyObjectWrapper) reference).getPythonObject();
+                }
+                references[i] = reference;
+            }
```

Comment:
I'm a _little_ bit worried about modifying the existing array, even if it probably doesn't matter in practice:

1. Caller _may_ want to do something w/ `references` after.
2. We are potentially wiping away the only reference to a `LivePyObjectWrapper`; if we get unlucky, would GC cause `LivePyObjectWrapper#destroy`, ie `PyObject#close`?

I think it's safer to create a new array.